.github/workflows/phpcs.yml

```diff
@@ -0,0 +1,17 @@
+name: PHPCS
```

Comment:
Hey, 

I got some questions on this file. I think we can skip the bot step. And its better to make use of the projects `.php_cs.dist.` I've made a [gist](https://gist.github.com/Tjoosten/68fb8dae16ac51cf21dd7aa2609eed39) as an example. Can u change the file contents with the content of this gist. Just for checking if its working. If yes i think that the gist is an cleaner fix. 

Kind regards, 
Tim Joosten

Reply:
Ohh, I didn't realize you wanted it to fix coding standard errors automatically. I usually shy away from those in case there's multiple fix available or the sorts. I'll try and get some time either today or tomorrow to change it then :+1:

Reply:
No problemo!